pkg/index/validate.go

```diff
@@ -20,10 +20,17 @@ import (
 	"strings"
 )
 
+const (
+	currentAPIVersion = "krew.googlecontainertools.github.com/v1alpha2"
```

Comment:
split in `groupName` and `groupVersion`: https://kubernetes.io/docs/concepts/overview/kubernetes-api/

Reply:
not sure how it will help since we're just doing a `==` check.

this can help do the split but IMO it's not necessary? https://github.com/kubernetes/apimachinery/blob/ed135c5b96450fd24e5e981c708114fbbd950697/pkg/runtime/schema/group_version.go#L216

Reply:
I was looking for that :). We can rely on a simple string check for now. We  
 have to change this later (at least after the next index file update).